### PR TITLE
Change base images from debian to alpine

### PR DESCRIPTION
**Pull request by klutchell**
Change-type: minor
Signed-off-by: Kyle Harding <kyle@balena.io>

```
[Info]     ┌─────────┬────────────┬─────────────────────┐
[Info]     │ Service │ Image Size │ Build Time          │
[Info]     ├─────────┼────────────┼─────────────────────┤
[Info]     │ core    │ 569.02 MB  │ 21 seconds          │
[Info]     ├─────────┼────────────┼─────────────────────┤
[Info]     │ worker  │ 414.44 MB  │ 1 minute, 9 seconds │
[Info]     └─────────┴────────────┴─────────────────────┘
```

### CODE REVIEW

**Comment by klutchell** [comment]
@jakogut I haven't tested on pi3 yet but this seems to have broken my qemu worker communication with the DUT. Did I accidentally remove something crucial to networking or do I just need to refresh my iptables rules?

**Comment by klutchell** [comment]
```
qemu: could not load PC BIOS '/usr/share/OVMF/OVMF_CODE.fd'
```
Might be resolved by rebasing on https://github.com/balena-os/leviathan/pull/548

File: worker/lib/workers/qemu.ts [hunk 1]
```diff
@@ -276,7 +285,7 @@ class QemuWorker extends EventEmitter implements Leviathan.Worker {
     const dnsmasqArgs = [
       `--interface=${bridgeName}`,
       `--dhcp-range=${this.qemuOptions.network.dhcpRange}`,
-      '--conf-file',
```
Comment (klutchell) [inline worker/lib/workers/qemu.ts:279]: @jakogut I'm still experimenting with this, but was there a purpose to this flag with no file?

Comment (klutchell) [reply worker/lib/workers/qemu.ts:289]: Disables loading the default configuration file.

File: worker/Dockerfile.template [hunk 2]
```diff
@@ -16,30 +17,30 @@ COPY bin bin
...
-WORKDIR /usr/app
-
-COPY config config
-COPY entry.sh .
+# hadolint ignore=DL3018
+RUN apk add --no-cache \
+  libusb-dev dbus-dev \
+  gstreamer-tools gst-plugins-base gst-plugins-bad gst-plugins-good \
+  bridge bridge-utils iproute2 dnsmasq iptables qemu-img qemu-system-x86_64 qemu-modules && \
+  apk add --arch x86_64 --root / --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/v3.12/community ovmf
```
Comment (klutchell) [inline worker/Dockerfile.template:29]: OVMF only has published packages for x86_64 and aarch64 so explicitly install the x86_64 version for now.

**Comment by jakogut** [comment]
@balena-ci rebase

File: worker/Dockerfile.template [hunk 3]
```diff
@@ -17,13 +17,13 @@ COPY bin bin
 
 RUN npm run build
 
-FROM balenalib/%%BALENA_ARCH%%-alpine-node:12.19.1-3.12-run-20201211
-
 ENV UDEV 1
+FROM balenalib/%%BALENA_ARCH%%-alpine:3.12-run-20211030
```
Comment (klutchell) [inline worker/Dockerfile.template:21]: @jakogut I shouldn't have approved this, it effectively makes udev unavailable in the runtime container and cannot flash SD cards in testbot.

**Comment by klutchell** [comment]
@balena-ci I self-certify!

**Pull request merged**